app/views/smart_proxies/show.html.erb

```diff
@@ -0,0 +1,73 @@
+<% title(_('Smart Proxy: %s') % @smart_proxy.to_label) %>
+<%= smart_proxy_title_actions(@smart_proxy, authorizer) %>
+<div id="proxy-show" class="row">
+  <ul id="proxy-tab" class="nav nav-tabs">
+    <li class="active"><a href="#properties" data-toggle="tab"><%= _('Overview') %></a></li>
+    <li><a href="#services" data-toggle="tab"><%= _('Services') %></a></li>
+  </ul>
+  <div id="proxy-tab-content" class="proxy-content tab-content">
+    <div class="tab-pane active in" id="properties">
+      <div class="col-md-6">
+        <div class="row">
+          <h4><%= _('Details') %></h4>
+        </div>
+        <div class="row">
+          <div class="col-md-4">
+            <%= _('Communication status') %>
+          </div>
+          <div class="col-md-8">
+            <span class="proxy-show-status" data-url="<%= ping_smart_proxy_path(@smart_proxy) %>">
+              <%= image_tag '/assets/spinner.gif' %>
+            </span>
+          </div>
+        </div>
+        <div class="row">
+          <div class="col-md-4">
+            <%= _('URL') %>
+          </div>
+          <div class="col-md-8">
+            <%= @smart_proxy.url %>
+          </div>
+        </div>
+        <div class="row">
+          <div class="col-md-4">
+            <%= _('Version') %>
+          </div>
+          <div class="col-md-8">
+            <span class="proxy-version" data-url="<%= version_smart_proxy_path(@smart_proxy) %>">
+              <%= image_tag '/assets/spinner.gif' %>
+            </span>
+          </div>
+        </div>
+        <div class="row">
+          <div class="col-md-4">
+            <%= _('Features') %>
+            <%= refresh_proxy_icon(@smart_proxy, authorizer) %>
+          </div>
+          <div class="col-md-8">
+            <%= @smart_proxy.features.to_sentence %>
+          </div>
+        </div>
+        <div class="row">
+          <h4><%= _('Locations') %></h4>
```

Comment:
Exactly, for example in non-Satellite context, this bit will not be much useful.

Idea: If these are widgets, user could decide what to put on, we could offer Locations, Organizations (improved) and things like associations via Subnet/Domain.